src/schema/index.js

```diff
@@ -120,6 +121,7 @@ const rootFields = {
   services: Services,
   show: Show,
   status: Status,
+  system_time: SystemTime,
```

Comment:
@alloy in https://github.com/artsy/metaphysics/pull/812 you had commented that this can/should be included as part of `sale`.

This _does_ seem like a 'first-class' model (in a stretch-y way) in that it's something that Gravity wholly owns/controls, and _can_ be useful in contexts independent of a sale, so I'm 👍 for adding it here, or under `system` if we think we'll have more of these 'kinds' of things, or something of that nature.